core/src/main/java/io/questdb/cutlass/auth/Authenticator.java

```diff
@@ -27,8 +27,10 @@
 import io.questdb.cairo.SecurityContext;
 import io.questdb.network.Socket;
 import io.questdb.std.Mutable;
+import io.questdb.std.ObjList;
 import io.questdb.std.QuietCloseable;
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 public interface Authenticator extends QuietCloseable, Mutable {
```

Comment:
It might be a good idea to extract common methods from `Authenticator` and `HttpAuthenticator` into a common interface. Currently the `Authenticator` is only used in PGWire and ILP/TCP, but not in HTTP which is weird considering its name. WDYT?

Reply:
done